### PR TITLE
fix mybatis build error 

### PR DESCRIPTION
**Pull request by Jzow**
#112 use bson error use type question

### CODE REVIEW

File: mybatis/Cargo.toml [hunk 1]
```diff
@@ -39,12 +39,12 @@ format_bson = ["mybatis-core/format_bson"]
...
 mybatis-util = { version = "2.0.0", path = "../mybatis-util", default-features = false}
-mybatis-sql = { version = "2.0.0", path = "../mybatis-sql", default-features = false, optional = true}
-mybatis-core = { version = "2.0.0", path = "../mybatis-core", default-features = false, optional = true}
+mybatis-sql = { version = "2.0.1", path = "../mybatis-sql", default-features = false, optional = true}
+mybatis-core = { version = "2.0.1", path = "../mybatis-core", default-features = false, optional = true}
 mybatis-macro = { version = "2.0.3", path = "../mybatis-macro", default-features = false, optional = true}
 
 ################################################################
-rbson = { version = "2.0.4", features = ["uuid-0_8", "chrono-0_4"] }
+rbson = { version = "2.0.4"}
```
Comment (Jzow) [inline mybatis/Cargo.toml]: This may affect macro adjustment

**Review by Jzow — COMMENTED** [review]
fix

**Review by xuhanfeng2219 — APPROVED** [review]

**Review by xuhanfeng2219 — APPROVED** [review]

**Review by xuhanfeng2219 — CHANGES_REQUESTED** [review]
agree change to jow's mind to fixed this bug

**Review by Jzow — COMMENTED** [review]
you have reviewed it again. Maybe you need to appro it

**Review by xuhanfeng2219 — APPROVED** [review]
agree change version

**Pull request merged**